### PR TITLE
flho-297 Changes to the obtain step.

### PR DESCRIPTION
**Pull request by bendiggle**
* Removed toggled text boxes for buy, temp-possession and wont-be-in-possession options,
	* Moved option acquire through other means to the bottom of the options,
	* Removed toggled fields from fields config and route config,
	* Removed translations for fields
	* Removed some tests that aren't needed any more

### CODE REVIEW

File: apps/new-dealer/fields/index.js [hunk 1]
```diff
@@ -126,56 +126,26 @@ module.exports = {
     mixin: 'checkbox-group',
     validate: 'required',
     options: [{
-      value: 'buy',
-      toggle: 'buy-details',
-      child: 'partials/obtain-buy'
+      value: 'buy'
```
Comment (joefitter) [inline apps/new-dealer/fields/index.js]: this can just be a string 'buy', doesn't need to be in an object

File: apps/new-dealer/fields/index.js [hunk 2]
```diff
@@ -126,56 +126,26 @@ module.exports = {
...
     }, {
-      value: 'temporary-possession',
-      toggle: 'temporary-details',
-      child: 'textarea'
+      value: 'temporary-possession'
```
Comment (joefitter) [inline apps/new-dealer/fields/index.js]: and this

File: apps/new-dealer/fields/index.js [hunk 3]
```diff
@@ -126,56 +126,26 @@ module.exports = {
...
     }, {
       value: 'manufacture'
```
Comment (joefitter) [inline apps/new-dealer/fields/index.js]: and this

File: apps/new-dealer/fields/index.js [hunk 4]
```diff
@@ -126,56 +126,26 @@ module.exports = {
...
     }, {
+      value: 'wont-take-possession'
```
Comment (joefitter) [inline apps/new-dealer/fields/index.js]: and this

**Comment by bendiggle** [comment]
@joefitter Updated, friend

**Comment by joefitter** [comment]
LGTM! merge if it passes

**Pull request merged**